packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -538,6 +540,16 @@ export interface WorkspaceExt {
     $onTextSearchResult(searchRequestId: number, done: boolean, result?: SearchInWorkspaceResult): void;
 }
 
+export interface TimelineExt {
+    $getTimeline(id: string, uri: string, options: theia.TimelineOptions, internalOptions?: theia.TimelineOptions): Promise<Timeline | undefined>;
+}
+
+export interface TimelineMain {
+    $registerTimelineProvider(id: string, label: string, scheme: string | string[]): Promise<void>;
+    $fireTimelineChanged(e: TimelineChangeEvent | undefined): Promise<void>;
```

Comment:
we never fire or process `undefined`, left over?

Reply:
removed the `undefined`